js/views/Wizard.jsx

```diff
@@ -200,7 +211,16 @@ export default class Wizard extends React.Component {
               <button type="button" className="btn btn-sm btn-primary" onClick={this.addStrip.bind(this)}><i className="fa fa-plus-circle"></i> Add Strip</button>
               &nbsp;
               <button type="button" className="btn btn-sm btn-primary" onClick={this.onNext.bind(this)}><i className="fa fa-arrow-right"></i> Next Step</button>
+
+              <div className="warning">
+                <span>
+                  {this.state.peakedWidth ?
+                  "Warning: the maximum width reached (" + currentWidth+ "\")" :
```

Comment:
Wow, completely missed that. Must've misinterpreted "limit the width" to be max. It'll be an easier flip